InvenTree/InvenTree/settings.py

```diff
@@ -1015,7 +1015,10 @@
 # Ensure that the ALLOWED_HOSTS do not contain any scheme info
 for i, host in enumerate(ALLOWED_HOSTS):
     if '://' in host:
-        ALLOWED_HOSTS[i] = host.split('://')[1]
+        ALLOWED_HOSTS[i] = host = host.split('://')[1]
+
+    if ':' in host:
+        ALLOWED_HOSTS[i] = host = host.split(':')[0]
```

Comment:
Question: What is the reason for this?

Reply:
Looks like this has bled through from https://github.com/inventree/InvenTree/pull/6871 - which is now merged.

Apparently the format of `ALLOWED_HOSTS` does not include protocol *or* port